src/Illuminate/Console/LaravelStyle.php

```diff
@@ -0,0 +1,57 @@
+<?php namespace Illuminate\Console;
+
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Application as ConsoleApplication;
+
+class LaravelStyle extends SymfonyStyle {
+
+	const MAX_LINE_LENGTH = 120;
+
+	private $input;
+	private $questionHelper;
+	private $progressBar;
+	private $lineLength;
+	
+		/**
```

Comment:
Yeah copied from Symfony, but this is WIP so please comment on CS after it's finished.